saleor/product/models.py

```diff
@@ -305,19 +305,18 @@ def sort_by_attribute(
         )
 
     def prefetched_for_webhook(self, single_object=True):
-        if single_object:
-            return self.prefetch_related(
-                "attributes__values",
-                "attributes__assignment__attribute",
-                "media",
-            )
-        return self.prefetch_related(
+        common_fields = (
             "attributes__values",
             "attributes__assignment__attribute",
-            "collections",
-            "variants__stocks__allocations",
             "media",
-            "category",
+            "variants__attributes__values",
+            "variants__attributes__assignment__attribute",
+            "variants__variant_media__media",
+        )
+        if single_object:
+            return self.prefetch_related(*common_fields)
+        return self.prefetch_related(
+            "collections", "variants__stocks__allocations", "category", *common_fields
```

Comment:
Why all `variants`' prefetches are in common list and this one is included only when calling for many object. Is it correct?